Document/0x05b-Basic-Security_Testing.md

```diff
@@ -279,102 +279,19 @@ Frida also provides bindings for various languages, including Python, C, NodeJS,
-
-```shell
-$ virtualenv drozer
-$ source drozer/bin/activate
-(drozer) $
-```
-
-You're now ready to install the required version of pyOpenSSL and build it against the OpenSSL headers installed previously. A typo in the source of the pyOpenSSL version Drozer prevents successful compilation, so you'll need to fix the source before compiling. Fortunately, ropnop has figured out the necessary steps and documented them in a [blog post](https://blog.ropnop.com/installing-drozer-on-os-x-el-capitan/ "ropnop Blog - Installing Drozer on OS X El Capitan").
-Run the following commands:
-
-```shell
-$ wget https://pypi.python.org/packages/source/p/pyOpenSSL/pyOpenSSL-0.13.tar.gz
-$ tar xzvf pyOpenSSL-0.13.tar.gz
-$ cd pyOpenSSL-0.13
-$ sed -i '' 's/X509_REVOKED_dup/X509_REVOKED_dupe/' OpenSSL/crypto/crl.c
-$ python setup.py build_ext -L/usr/local/opt/openssl/lib -I/usr/local/opt/openssl/include
-$ python setup.py build
-$ python setup.py install
-```
-
-With that out of the way, you can install the remaining dependencies.
-
-```shell
-$ easy_install protobuf==2.4.1 twisted==10.2.0
-```
-
-Finally, download and install the Python .egg from the MWR labs website:
-
-```shell
-$ wget https://github.com/mwrlabs/drozer/releases/download/2.3.4/drozer-2.3.4.tar.gz
-$ tar xzf drozer-2.3.4.tar.gz
-$ easy_install drozer-2.3.4-py2.7.egg
-```
-
-###### Installing the Agent
-
-Drozer agent is the software component that runs on the device itself. Download the latest Drozer Agent [here](https://github.com/mwrlabs/drozer/releases/) and install it with adb.
-
-```shell
-$ adb install drozer.apk
-```
-
-###### Starting a Session
-
-You should now have the Drozer console installed on your host machine and the Agent running on your USB-connected device or emulator. Now you need to connect the two to start exploring.
-
-Open the Drozer application in the running emulator and click the OFF button at the bottom of the app to start an Embedded Server.
-
-![alt text](Images/Chapters/0x05b/server.png "Drozer")
-
-The server listens on port 31415 by default. Use adb to forward this port to the localhost interface, then run Drozer on the host to connect to the agent.
-
-```shell
-$ adb forward tcp:31415 tcp:31415
-$ drozer console connect
-```
+###### Basic Drozer Commands
 
- Use the "list" command to view all Drozer modules that can be executed in the current session.
+Once the setup is completed you can start a session to an emulator or a device connected per USB by running `adb forward tcp:31415 tcp:31415` and `drozer console connect`. See the full instructions [here](https://mobiletools.mwrinfosecurity.com/Starting-a-session/ "Starting a Session").
 
-###### Basic Drozer Commands
+Here's a non-exhaustive list of commands you can use to start exploring on Android:
 
 - To list all the packages installed on the emulator, execute the following command:
```

Comment:
"To list all the packages installed on the emulator" -> To list all installed packages 
Just referring to the emulator would otherwise be misleading

Reply:
right! done